### ISSUE TITLE
Gentoo Ebuild Proposal for Thorium Browser

### ISSUE THREAD

**Issue by nnzv** [Alex313031/thorium#357]
I propose creating a Gentoo ebuild for the Thorium browser. While I lack experience with Chromium-based ebuilds, I'm willing to try. If anyone is currently working on this, or in the future, consider the official [Chromium ebuild](https://github.com/gentoo/gentoo/tree/master/www-client/chromium) in the Gentoo repository as a resource.

**Comment by theoparis** [comment]
I would love to see this.

Also, It'd be interesting there was a use flag for said ebuild to enable the ChromiumOS WM/DE via [ChromiumOS on Linux](https://chromium.googlesource.com/chromium/src/+/HEAD/docs/chromeos_build_instructions.md#Chromium-OS-on-Linux-linux_chromeos).

**Comment by radeaosk** [comment]
yes, I'd also like to see Thorium on gentoo. I'll post an update if I make any progress on this myself

**Comment by GrabbenD** [comment]
This would be awesome

@radeaosk Did you make any progress?

**Comment by fraschm1998** [comment]
> This would be awesome!
> 
> Here's a duplicate [thread](https://forums.gentoo.org/viewtopic-p-8816628.html?sid=dcda93c1d55e8fa65b03d4f0a2621a89) on Gentoo's forum. There's also this [Cromite EBuild](https://github.com/PF4Public/gentoo-overlay/tree/master/www-client/cromite) which could probably be re-purposed (but like you said @nnzv, the same should be doable with the official [Chromium EBuild](https://packages.gentoo.org/packages/www-client/chromium)).
> 
> @radeaosk Did you make any progress?

just made it: https://github.com/gentoo/gentoo/pull/36882

![image](https://github.com/Alex313031/thorium/assets/54919428/a4c8bc4a-fc86-491d-b440-c939cb053135)

**Comment by nnzv** [comment]
@fraschm1998, I noticed that your PR was rejected. I suggest considering the maintainer's advice by either contributing to the guru ebuild repository or creating your own repository to share your work with others.

**Comment by fraschm1998** [comment]
> @fraschm1998, I noticed that your PR was rejected. I suggest considering the maintainer's advice by either contributing to the guru ebuild repository or creating your own repository to share your work with others.

Just submitted a pull request to Guru repo [here](https://github.com/gentoo-mirror/guru/pull/22).

**Comment by blackteahamburger** [comment]
I am curious about why no one can write a source-based ebuild...

**Comment by nnzv** [comment]
It sounds like it was a lot to handle, buddy. At least, that’s what I think.

**Comment by blackteahamburger** [comment]
Made one in gentoo-zh: https://github.com/microcai/gentoo-zh/tree/master/www-client/thorium-browser-bin

I dislike the complicated process to submit ebuilds to GURU, so I submitted it to a repository that I'm familiar with.

**Comment by blackteahamburger** [comment]
Done! I'm now inviting everyone to test in https://github.com/blackteahamburger/my-overlay, especially for those using mold to see whether rui314/mold#416 is reproducible.

**Comment by gz83** [comment]
Thorium currently lacks the manpower and resources to do this.
